user/reference/overview.md

```diff
@@ -262,6 +262,27 @@ Each tier of instance size delivers more vCPU and RAM resources available at you
 | 2x-large  | 16          | ~64         | requires credits to use; may be limited to certain Plans|
 
 
+## GPU VM Instance Size 
+
+You can choose to run your GPU builds on larger instances size. All you need to do is type the appropriate instance size to your `.travis.yml` using the following tags and one of the available values:
```

Comment:
Can I suggest:


You can choose to run your GPU builds on various GPU "sizes".